packages/wallet/src/PersonalWallet/PersonalWallet.ts

```diff
@@ -451,6 +461,17 @@ export class PersonalWallet implements ObservableWallet {
       }),
       stores.assets
     );
+
+    this.handles$ = handlePolicyIds?.length
+      ? createHandlesTracker({
+          assetInfo$: this.assetInfo$,
+          handlePolicyIds,
+          logger: contextLogger(this.#logger, 'handles$'),
+          tip$: this.tip$,
+          utxo$: this.utxo.total$
+        })
+      : throwError(() => new InvalidConfigurationError('Missing handlePolicyIds option in PersonalWallet'));
```

Comment:
Does this mean that specifying `handlePolicyIds` in the constructor is mandatory?